nuclia/sdk/search.py

```
@@ -192,23 +192,37 @@ def ask_json(
         See https://docs.nuclia.dev/docs/api#tag/Search/operation/Ask_Knowledge_Box_kb__kbid__ask_post
         """
         ndb: NucliaDBClient = kwargs["ndb"]
+        if isinstance(schema, str):
+            if os.path.exists(schema):
```

what if the file does not exist? should we raise an error?